<launch>

    <!-- =================================================================== -->
    <!-- Vehicle ID Number -->
    <!-- =================================================================== -->
    <rosparam param="/vehicle_id"> 2 </rosparam>

    <!-- =================================================================== -->
    <!-- Console Logging Settings -->
    <!-- =================================================================== -->
    <rosparam param="/logging/console/enable"> true </rosparam>
    <rosparam param="/logging/console/log_node"> true </rosparam>
    <rosparam param="/logging/console/log_data"> true </rosparam>
    <rosparam param="/logging/console/log_debug"> true </rosparam>
    <rosparam param="/logging/console/log_info"> true </rosparam>
    <rosparam param="/logging/console/log_warning"> true </rosparam>
    <rosparam param="/logging/console/log_error"> true </rosparam>

    <!-- =================================================================== -->
    <!-- File Logging Settings -->
    <!-- =================================================================== -->
    <rosparam param="/logging/file/enable"> false </rosparam>
    <rosparam param="/logging/file/log_node"> true </rosparam>
    <rosparam param="/logging/file/log_data"> true </rosparam>
    <rosparam param="/logging/file/log_debug"> true </rosparam>
    <rosparam param="/logging/file/log_info"> true </rosparam>
    <rosparam param="/logging/file/log_warning"> true </rosparam>
    <rosparam param="/logging/file/log_error"> true </rosparam>

    <!-- =================================================================== -->
    <!-- Nodes -->
    <!-- =================================================================== -->

    <node name="tcp_server_test_node" pkg="avl_asio" type="tcp_server_test_node"
        required="true" clear_params="true" output="screen">
        <rosparam command="load" file="/var/avl_config/tcp_server_test_node.config"/>
    </node>

    <node name="tcp_socket_test_node" pkg="avl_asio" type="tcp_socket_test_node"
        required="true" clear_params="true" output="screen">
        <rosparam command="load" file="/var/avl_config/tcp_socket_test_node.config"/>
    </node>

    <!-- <node name="udp_socket_test_node1" pkg="avl_asio" type="udp_socket_test_node"
        required="true" clear_params="true" output="screen">
        <rosparam command="load" file="/var/avl_config/udp_socket_test_node1.config"/>
    </node>

    <node name="udp_socket_test_node2" pkg="avl_asio" type="udp_socket_test_node"
        required="true" clear_params="true" output="screen">
        <rosparam command="load" file="/var/avl_config/udp_socket_test_node2.config"/>
    </node> -->

</launch>
